rnd/infra/terraform/.terraform.lock.hcl

```diff
@@ -0,0 +1,22 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
Should we be committing this file?

Reply:
yup, we need these to maintain versions. probably one thing we could do is also lockdown platform specific - but I didn't cause ideally we'll do terraforming via CI and never on our own machines